src/scss/_brand.scss

```diff
@@ -10,57 +10,66 @@
     @return oBrandSupportsVariant($component: 'o-header-services', $variant: $variant);
 }
 
-@include oBrandDefine('o-header-services', 'master', (
+$brand: oBrandGetCurrentBrand();
+
+@if $brand == 'master' {
+	@include oBrandDefine('o-header-services', 'master', (
 	'variables': (
-		logo: 'brand-ft-logo-square-coloured',
-		top-text: oColorsGetPaletteColor('white'),
-		top-background: oColorsGetPaletteColor('black'),
-		top-link: oColorsGetPaletteColor('white'),
-		top-link-hover: oColorsGetPaletteColor('black-20'),
-		product-title: oColorsGetPaletteColor('white'),
-		nav-text: oColorsGetPaletteColor('black'),
-		nav-background: oColorsGetPaletteColor('white-60'),
-		nav-border: oColorsGetPaletteColor('black-20'),
-		nav-hover-text: oColorsGetPaletteColor('black'),
-		nav-hover-background: oColorsGetPaletteColor('black-5'),
-		subnav-text: oColorsGetPaletteColor('black'),
-		subnav-background: oColorsGetPaletteColor('white-40'),
-		'b2b': (
-			product-title: oColorsGetPaletteColor('white'),
-			nav-text: oColorsGetPaletteColor('white'),
-			nav-background: oColorsGetPaletteColor('claret'),
-			nav-hover-text: oColorsGetPaletteColor('white'),
-			nav-hover-background: oColorsGetPaletteColor('claret-30')
-		),
-		'b2c': (
-			product-title: oColorsGetPaletteColor('org-b2c-light'),
-			nav-text: oColorsGetPaletteColor('white'),
-			nav-background: oColorsGetPaletteColor('org-b2c'),
-			nav-hover-text: oColorsGetPaletteColor('white'),
-			nav-hover-background: oColorsGetPaletteColor('org-b2c-dark')
-		)
+	logo: 'brand-ft-logo-square-coloured',
```

Comment:
If we're opening up oBrand soon to other users do you think the number of variables could be consolidated? E.g. maybe `top-text` and `product-title` could be something along the lines of `foreground-color`?